modulemd/tests/test-modulemd-modulestream.c

```diff
@@ -1021,6 +1021,21 @@ module_stream_test_v2_yaml (void)
   g_clear_pointer (&specV2Path, g_free);
 }
 
+
+static void
+module_packager_v2_sanity (void)
+{
+  g_autoptr (ModulemdModuleStream) stream = NULL;
+  g_autoptr (GError) error = NULL;
+
+  g_autofree gchar *specV2Path = g_strdup_printf (
+    "%s/yaml_specs/modulemd_stream_v2.yaml", g_getenv ("MESON_SOURCE_ROOT"));
```

Comment:
Did you intend this to be `modulemd_packager_v2.yaml`? Otherwise it's completely redundant with the very last check in the function above.

Reply:
Whoops... yes I did.